src/config.rs

```diff
@@ -214,13 +214,15 @@ impl ConfigHelpItem {
 }
 
 macro_rules! create_config {
-    ($($i:ident: $ty:ty, $def:expr, $( $dstring:expr ),+ );+ $(;)*) => (
+    ($($i:ident: $ty:ty, $def:expr, $exp:expr, $( $dstring:expr ),+ );+ $(;)*) => (
         #[derive(Clone)]
         pub struct Config {
+            // are we using unstable features or not
+            unstable_features: bool,
```

Comment:
Could we use a macro generated option for this like the other options? I think that would simplify things a bit